projects/eslint-config/plugins/liferay/lib/rules/no-abbreviations.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * SPDX-FileCopyrightText: © 2017 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: MIT
+ */
+
+const ABBREVIATIONS_REPLACEMENTS = {
+	arr: ['array'],
+	btn: ['button'],
+	cb: ['callback'],
+	desc: ['description'],
+	e: ['event', 'error'],
+	el: ['element'],
+	err: ['error'],
+	evt: ['event'],
+	fm: ['form'],
+	fmt: ['format'],
+	idx: ['index'],
+	img: ['image'],
+	lg: ['large'],
+	md: ['medium'],
+	obj: ['object'],
+	opts: ['options'],
+	prj: ['project'],
+	sm: ['small'],
+};
+
+const formatMessage = function (abbreviation, replacements) {
+	if (replacements.length > 1) {
+		return `Avoid ${abbreviation} abbreviation, preferred alternatives are [${replacements.join(
+			','
+		)}]`;
+	}
+	else {
+		return `Avoid ${abbreviation} abbreviation, preferred alternative is ${replacements}`;
```

Comment:
Since the abbreviation and replacement is dynamic based on the error, let's wrap it in quotes so that it stands out to the reader.

```suggestion
		return `Avoid "${abbreviation}" abbreviation, preferred alternative is "${replacements}"`;
```